app/frontend/services/api/index.ts

```diff
@@ -262,6 +272,52 @@ export class Api {
     return this.client.patch<ApiResponse<IPermitApplication>>(`/permit_applications/${id}/update_version`)
   }
 
+  async assignCollaboratorToPermitApplication(
+    permitApplicationId: string,
+    params: {
+      collaboratorId: string
+      collaboratorType: ECollaboratorType
+      assignedRequirementBlockId?: string
+    }
+  ) {
+    return this.client.post<ApiResponse<IPermitCollaboration>>(
+      `/permit_applications/${permitApplicationId}/permit_collaborations`,
+      {
+        permitCollaboration: params,
+      }
+    )
+  }
+
+  async inviteNewCollaboratorToPermitApplication(
+    permitApplicationId: string,
+    params: {
+      user: {
+        email: string
+        firstName: string
+        lastName: string
+      }
+      collaboratorType: ECollaboratorType
+      assignedRequirementBlockId?: string
+    }
+  ) {
+    return this.client.post<ApiResponse<IPermitCollaboration>>(
+      `/permit_applications/${permitApplicationId}/permit_collaborations/invite`,
+      {
+        collaboratorInvite: params,
+      }
+    )
+  }
+
+  async unassignPermitCollaboration(id: string) {
+    return this.client.delete<ApiResponse<IPermitCollaboration>>(`/permit_collaborations/${id}`)
+  }
+
+  async reInvitePermitCollaboration(permitCollaborationId: string) {
```

Comment:
"reinvite" may match the rest of the app but maybe you want it to be searced separately

Reply:
good catch